src/inputs/DateField.tsx

```diff
@@ -32,6 +32,8 @@ export interface DateFieldProps
   placeholder?: string;
   format?: keyof typeof dateFormats;
   iconLeft?: boolean;
+  // Set custom logic for individual dates or date ranges to be disabled in the picker, exposed from `react-day-picker`
+  disabledDays?: Modifier;
```

Comment:
**thought:** What do you think about adding a link to the docs for what a Modifier can be?